### PR TITLE
feat: email template model, migration script for marketing templates

### PR DESCRIPTION
**Pull request by chenxuan-zhou**
…g email templates

### Tickets:

-   [HCK-17](https://linear.app/hackmcgill/issue/HAC-17/move-email-templates-to-database)

### List of changes:

- added email template model
- created a script to move [existing templates](https://github.com/hackmcgill/hackerAPI/tree/dev/assets/email/marketingEmail) to database

## Type of change

-   [ ] New feature (non-breaking change which adds functionality)

### How has this been tested?

-   [ ] Ran `NODE_ENV=test && node ./scripts/migrate_email_templates.js` manually

**Test Configuration**:

**Firmware version:**
**Hardware:**
**Toolchain:**
**SDK:**

### Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] My changes generate no new warnings
-   [ ] Listed change(s) in the Changelog
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] I have made corresponding changes to the documentation
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: scripts/migrate_email_templates.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+"use strict";
+const Services = {
+    env: require("../services/env.service"),
+    db: require("../services/database.service")
+}
```
Comment (jamesxu123) [inline scripts/migrate_email_templates.js:5]: Do we usually do requires inside an object? If not, I think it's a bit more intuitive to just say
```
const env = require()
const db = require()
```

File: scripts/migrate_email_templates.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Inserts an EmailTemplate document to the db. Prevents duplicate name.
+ * @param {EmailTemplate} emailTemplateDoc 
+ */
+async function insertOne(emailTemplateDoc) {
+    const dup = await EmailTemplate.collection.findOne({ name: emailTemplateDoc.name });
+    if (!dup) {
+        console.log(`Migrating ${emailTemplateDoc.name}`);
+        await EmailTemplate.collection.insertOne(emailTemplateDoc);
+    }
```
Comment (jamesxu123) [inline scripts/migrate_email_templates.js:67]: I think we should log something explicit if there are duplicates instead of just silently failing

Comment (chenxuan-zhou) [reply scripts/migrate_email_templates.js]: I have added a `console.error()` message in commit [7f37e97](https://github.com/hackmcgill/hackerAPI/commit/7f37e97cf6463a7b646d3837fb91ffed3a52ae0f).

**Review by jamesxu123 — APPROVED** [review]
LGTM

**Pull request merged**